### PR TITLE
[PCFG-43] DB Connection in health-check

### PR DESCRIPTION
**Pull request by aomegax**
Added info about db connection to health-check service.

#### List of Changes
- db connection status:
  - "up": db connection established properly
  - "down": db connection lost

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by jacopocarlini — CHANGES_REQUESTED** [review]

File: src/main/java/it/pagopa/pagopa/apiconfig/repository/HealthCheckRepository.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package it.pagopa.pagopa.apiconfig.repository;
```
[the remark this answers is not in the document]
Comment (aomegax) [reply src/main/java/it/pagopa/pagopa/apiconfig/repository/HealthCheckRepository.java:1]: done

File: src/main/resources/application-h2.properties [hunk 2]
```diff
@@ -21,6 +21,9 @@ spring.jpa.hibernate.use-new-id-generator-mappings=false
 spring.jpa.hibernate.ddl-auto=none
 spring.jpa.defer-datasource-initialization=true
 
+# HikariCP configuration
```
Comment (pasqualespica) [inline src/main/resources/application-h2.properties:24]: not needed [connectionTimeout](https://github.com/brettwooldridge/HikariCP#frequently-used) sets to 30s
_This property controls the maximum number of milliseconds that a client (that's you) will wait for a connection from the pool. If this time is exceeded without a connection becoming available, a SQLException will be thrown. Lowest acceptable connection timeout is 250 ms. Default: 30000 (30 seconds)_

File: src/main/java/it/pagopa/pagopa/apiconfig/controller/HomeController.java [hunk 3]
```diff
@@ -55,10 +60,12 @@ public RedirectView home() {
 
     @GetMapping("/info")
     public ResponseEntity<AppInfo> healthCheck() {
```
Comment (pasqualespica) [inline src/main/java/it/pagopa/pagopa/apiconfig/controller/HomeController.java:62]: couldn't we do it with [actuator](https://www.baeldung.com/spring-boot-health-indicators#3-additional-details) ?

**Review by pasqualespica — APPROVED** [review]

File: src/main/resources/application-h2.properties (hunk 2 above)
Comment (aomegax) [reply src/main/resources/application-h2.properties:24]: yes, it is a default parameter, I set it only as reminder to highlight difference with the other properties files.

File: src/main/java/it/pagopa/pagopa/apiconfig/controller/HomeController.java (hunk 3 above)
Comment (aomegax) [reply src/main/java/it/pagopa/pagopa/apiconfig/controller/HomeController.java:62]: as said in other meeting, to adopt actuator properly, we should define who and how will consume info.
for time restriction of the project we could assume this way is preferable to reach the goal.

**Review by jacopocarlini — APPROVED** [review]
great

**Pull request merged**